db/schema.rb

```diff
@@ -592,7 +602,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "scanner_working"
-    t.index ["uploader_type", "uploader_id"], name: "index_malware_scan_results_on_uploader"
+    t.index ["uploader_type", "uploader_id"], name: "index_malware_scan_results_on_uploader_type_and_uploader_id"
```

Comment:
As above.

Reply:
Thanks @mpw5, I also don't understand how the unrelated schema changes crept in (I definitely pulled the version from main before I started!) but your suggestion removed them anyway.